src/render/tutorials/caching-http-requests/2_0.html.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Caching HTTP requests
+author: oskarjerzyk
+keywords: tutorial
+date: 2019-10-21
+layout: tutorial
+knotxVersions:
+    - 2.0.0
+---
+# Overview
+In this tutorial we will add ability to cache http request to project based on [Knot.x Starter Kit](https://github.com/Knotx/knotx-starter-kit) 
+template.
+
+We will use existing `ActionFactory` from [knotx-fragments](https://github.com/Knotx/knotx-fragments) to achieve our goal
+which will be implementing custom endpoint with cache ability.
+
+## Prerequisites
+- JDK 8
+- Docker
+
+## What you're going to learn:
+- How to add cache ability to API
+- How to set up Wiremock (we don't need to implement real API, mock API will be enough for our training purposes)
+- How to deploy project to Docker Swarm
+
+Download the [Latest Knot.x Starter Kit release](https://github.com/Knotx/knotx-starter-kit/releases) and unzip it.
+
+Project has the following structure:
+```
+├── docker
+|   ├── Dockerfile                // Docker file with image definition.
+├── functional                    // Keep here your functional tests. Example implementation included
+├── gradle                        // Gradle wrapper and common gradle scripts
+├── knotx/conf                    // Knotx configuration which will be copied to docker image
+├── modules                       // Sub-modules of your project
+│   ├── ...                       // example modules implementation
+```
+
+## Project configuration
+The first thing we need to do is to specify configuration files. Let's start in the project level directory and simply create `api-cache.yml`
```

Comment:
What do you think about this oder:
- I use Configurable Integrations so I want to use tasks
- I configure HTTP action to invoke my target API
- then I configure in-memory cache and explain that it can be exchange with Redis very easily
- now specify task and and configure Open API / operation and Handlers
- build image, it is ready to use
- no we need to provider an environment, we choose Docker Swarm
- for target API we use Wiremock and its configuration looks like...
- let's start Docker Swarm and open in browser http://localhostL8092/..., refresh a few times
- end.

Reply:
fixed